app/views/admin/products_v3/_product_row.html.haml

```diff
@@ -40,7 +40,7 @@
 %td.align-right
   = render(VerticalEllipsisMenu::Component.new) do
     = link_to t('admin.products_page.actions.edit'), edit_admin_product_path(product), 'data-turbo': false
-    = link_to t('admin.products_page.actions.clone'), clone_admin_product_path(product), 'data-turbo': false
+    = link_to t('admin.products_page.actions.clone'), admin_clone_product_path(product), 'data-turbo-method': :post
```

Comment:
This _looks_ simple, but I'm wondering how hard it was to find the exact right name for `'data-turbo-method': :post`? Maybe we need to document this somewhere 🤔

Reply:
Yes, this was the only stimulus-related takeaway from this. Sure, will document this one 👍🏻